nunaliit2-couch-application/src/main/atlas_couchapp/_attachments/tools/index.html

```diff
@@ -25,6 +25,7 @@ <h1>Nunaliit2 Tools</h1>
 			<li class="n2tools_item_import"><a href="./import.html">Import Tool</a></li>
 			<li class="n2tools_item_debug"><a href="./debug.html">Debug Tool</a></li>
 			<li class="n2tools_item_guitool"><a href="./schema_editor.html">Schema Editor</a></li>
+			<li class="n2tools_item_metadata"><a href="metadata.html">Metadata Editor</a></li>
```

Comment:
Missing relative path.
```suggestion
			<li class="n2tools_item_metadata"><a href="./metadata.html">Metadata Editor</a></li>
```

Reply:
Fixed.